authorization-service/.env

```diff
@@ -0,0 +1 @@
+dilbarakkaya=TEST_PASSWORD
```

Comment:
Please do not commit .env file (add .env in the gitignore)

.env file is used for keeping secrets on your machine and not for public usage

Reply:
done